src/js/langs/nb_NO.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview I18N for Norwegian
+ * @author Anton Reytarovskiy <reitarovskii.toh@gmail.com>
+ */
+import i18n from '../i18n';
+
+i18n.setLanguage(['nb', 'nb-NO'], {
+    'Markdown': 'Funksjonaliteter',
+    'WYSIWYG': 'WYSIWYG',
+    'Write': 'Skriv',
+    'Preview': 'Forhåndsvisning',
+    'Headings': 'Overskrift',
+    'Paragraph': 'Paragraf',
+    'Bold': 'Fet skrift',
+    'Italic': 'Italic',
+    'Strike': 'Strike',
+    'Code': 'Kode',
+    'Line': 'Linje',
+    'Blockquote': 'Blokksitat',
+    'Unordered list': 'Usortert liste',
+    'Ordered list': 'Sortert liste',
+    'Task': 'Task',
+    'Indent': 'Indent',
+    'Outdent': 'Outdent',
+    'Insert link': 'Sett inn lenke',
+    'Insert CodeBlock': 'Sett inn CodeStreng',
+    'Insert table': 'Sett inn diagram',
+    'Insert image': 'Sett inn bilde',
+    'Heading': 'Overskrift',
+    'Image URL': 'BildeURL',
+    'Select image file': 'Velg bildefil',
+    'Description': 'Beskrivelse',
+    'OK': 'OK',
+    'More': 'Mer',
+    'Cancel': 'Angre',
+    'File': 'Fil',
+    'URL': 'URL',
+    'Link text': 'Lenketekst',
+    'Add row': 'Legg til linje',
+    'Add col': 'Legg til kolonne',
+    'Remove row': 'Fjern rad',
+    'Remove col': 'Fjern kolonne',
+    'Align left': 'Venstreorienter',
+    'Align center': 'Senterorienter',
+    'Align right': 'Høyreorienter',
+    'Remove table': 'Fjern diagram',
+    'Would you like to paste as table?': 'Ønsker du å lime inn som et diagram?',
+    'Text color': 'Tekstfarge',
+    'Auto scroll enabled': 'Auto-scroll aktivert',
+    'Auto scroll disabled': 'Auto-scroll deaktivert',
+    'Choose language': 'Velg språl'
```

Comment:
Code convention is not right. Running the command below shows that the [indentation is incorrect](https://eslint.org/docs/rules/indent) in file `nb_NO.js`.
``` sh
npm run lint
```
![error](https://user-images.githubusercontent.com/18183560/71337809-7a900a80-2590-11ea-984e-14971af5fd77.png)

Our code convention is indent `2`.